recirq/benchmarks/rep_rate/rep_rate_calculator.py

```diff
@@ -119,19 +119,23 @@ def __init__(self, device: cirq.Device, sampler: cirq.Sampler,
             sampler: Sampler to test for rep rate.
             gate: two-qubit gate to use for sample circuits.
         """
+        if device.metadata is None:
+            raise ValueError("Invalid device: device must contain metadata.")
+
         self.device = device
         self.sampler = sampler
         self.gate = gate
-        self.qubits = list(self.device.qubits)
+        self.qubits = sorted(self.device.metadata.qubit_set) if self.device.metadata is not None else []
```

Comment:
isn't the `if` redundant here because of the valueerror?

Reply:
I thought that as well, but mypy still complained...

Reply:
I think it is getting confused between self.device and device.  I bet if you reorder things a bit, mypy won't complain.

Reply:
aha that was it, thanks for the catch